node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala

```diff
@@ -53,8 +54,8 @@ class NarrowTransactionGenerator(
     val generated = (0 until (n * 1.2).toInt).foldLeft(
       (
         Seq.empty[Transaction],
-        preconditions.tradeAsset.fold(Seq.empty[IssueTransactionV2])(Seq(_)),
-        preconditions.tradeAsset.fold(Seq.empty[IssueTransactionV2])(Seq(_)),
+        preconditions.tradeAsset.fold(Seq.empty[IssueTransaction])(Seq(_)),
+        preconditions.tradeAsset.fold(Seq.empty[IssueTransaction])(Seq(_)),
         Universe.Leases,
         Seq.empty[CreateAliasTransaction]
```

Comment:
Sorry, I do not understand these pages. Do you have any assistance or guide to understand the programming words,,such as timestamp, val, cats show, final case class, etc. (especially 51 is hard to understand)